modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/TargetRequest.java

```diff
@@ -394,6 +393,8 @@ public int write(NHttpClientConnection conn, ContentEncoder encoder) throws IOEx
     public boolean hasEntityBody() {
         return hasEntityBody;
     }
+
+    public URL getUrl(){return url; }
     
```

Comment:
Shall we remove the extra space here?

Reply:
fixed in https://github.com/wso2/wso2-synapse/pull/1305